cicecore/drivers/unittest/bcstchk/bcstchk.F90

```diff
@@ -0,0 +1,281 @@
+
+      program bcstchk
```

Comment:
what is being checked?  I'm guessing bc = boundary condition.  st = ?  Maybe there's some documentation that I haven't gotten to yet...

Reply:
ice_broadcast.F90 methods.  I am updating the documentation.